apacheconfig/loader.py

```diff
@@ -288,6 +283,27 @@ def _merge_contents(self, contents, items):
 
         return contents
 
+    def _merge_dicts(self, dict1, dict2):
+        "merges dict2 into dict1"
+        for key in dict2:
+            if key in dict1:
+                if isinstance(dict1[key], dict) and isinstance(dict2[key], dict):
+                    self._merge_dicts(dict1[key], dict2[key])
+                elif dict1[key] == dict2[key]:
+                    pass # same leaf value
+                else:
+                    if self._options.get('allowmultioptions', True):
+                        if not isinstance(dict1[key], list):
+                            dict1[key] = [dict1[key]]
+                        if not isinstance(dict2[key], list):
+                            dict2[key] = [dict2[key]]
+                        dict1[key] = list(set(dict1[key]).union(dict2[key]))
```

Comment:
Conversion into `set` looses ordering of the list elements. I feel like this can be a problem because order of options may be important...